modin/experimental/xgboost/xgboost.py

```diff
@@ -126,29 +141,36 @@ def train(
     evals_result: Optional[Dict] = None,
     **kwargs,
 ):
-    """
-    Train XGBoost model.
+    """Run distributed training of XGBoost model.
+
+    During work it evenly distributes `dtrain` between workers according
+    to IP addresses partitions (in case of not evenly distribution of `dtrain`
+    is possible by IPs, part of partitions will be re-distributed between nodes),
```

Comment:
```suggestion
    to IP addresses partitions (in case of not even distribution of `dtrain`
    over nodes, some partitions will be re-distributed between nodes),
```
I think this looks more correct

Reply:
applied